test/Turn-test.js

```diff
@@ -0,0 +1,56 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Turn = require('../src/Turn');
+const Card = require('../src/Card')
+
+let turn;
+let turn2;
+let turn3;
+let card;
+
+describe('Turn', function () {
+ beforeEach(function() {
+    turn = new Turn()
+    turn2 = new Turn('pug')
+    turn3 = new Turn('pug', card)
+    card = new Card(1,'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object')
+ })
+it('should be a function', function() {
+  expect(Turn).to.be.a("function");
+})
+
+it('should be an instance of Turn', function() {
+  expect(turn).to.be.an.instanceOf(Turn);
+})
+
+it('should store a users guess', function() {
+  expect(turn2.userGuess).to.equal('pug')
+})
+
+it('should store a Card object', function() {
+  expect(turn2.userGuess).to.equal('pug')
+  expect(turn3.card).to.deep.equal(card)
+})
+
+it('should return a users guess', function() {
+  expect(turn2.returnGuess()).to.equal('pug')
+})
+
+it('should return a card', function() {
```

Comment:
I need to look at this again thinking to add another expect sense it fails when I try to use the turn3 global variable in the chai.